docs/intro/install.rst

```diff
@@ -223,6 +224,28 @@ After any of these workarounds you should be able to install Scrapy::
   pip install Scrapy
 
 
+PyPy
+----
+
+We recommend using the latest PyPy version. The version tested is PyPy2-v5.9.0.
+
+Most scrapy dependencides now have binary wheels for CPython, but not for PyPy.
+This means that these dependecies will be built during installation.
+On OS X, you are likely to face an issue with building Cryptography dependency,
+solution to this problem is described
+`here <https://github.com/pyca/cryptography/issues/2692#issuecomment-272773481>`_,
+that is to ``brew install openssl`` and then export the flags that this command
+recommends (only needed when installing scrapy). Installing on Linux has no special
+issues besides installing build dependencies.
+Installing scrapy with PyPy on Windows is not tested.
+
+You can check that scrapy is installed correctly by running ``scrapy bench``.
+If this command gives errors such as
+``TypeError: ... got 2 unexpected keyword arguments``, this means
+that setuptools was unable to pick up one PyPy-specific dependency.
+To fix this issue, run ``pip install 'PyPyDispatcher>=2.1.0'``.
```

Comment:
@lopuhin do you prefer to keep this note just in case, or to remove it?

Reply:
I'd rather keep it just in case. In theory pypy comes with a recent enough setuptools, but I remember that someone still had this issue somehow (although that could be with 1.4.0 which I thought already had this feature). If we don't get any reports about this, we can remove it in the next release.